packages/datetime/package.json

```diff
@@ -24,6 +24,7 @@
     "test": "npm-run-all -s compile:cjs test:pre -p test:karma test:iso",
     "test:pre": "tsc -p ./test",
     "test:karma": "karma start",
+    "test:karma:debug": "karma start --browsers=Chrome --single-run=false --reporters=mocha --debug",
```

Comment:
Can we get rid of `--browsers=Chrome` for this task? I've been changing the browser to "PhantomJS" locally/manually for the past few months whenever I need to write tests. It's much more useful to dev against the environment that Circle is actually going to use. 

Would be cool to have a `test:karma:debug:chrome` task as a supplement though.

Reply:
can be a follow up

Reply:
actually I prefer your approach, will remove `--browsers` and require devs to supply that override manually